packages/gui/src/components/offers2/OfferBuilderViewer.tsx

```diff
@@ -308,7 +304,7 @@ function OfferBuilderViewer(props: OfferBuilderViewerProps, ref: any) {
                 canCounter={canCounterOffer}
```

Comment:
I think canCounter should be removed here?

Reply:
Thanks, fixed